xbmc/favourites/ContextMenus.h

```diff
@@ -20,34 +20,34 @@ class CFavouriteContextMenuAction : public CStaticContextMenuAction
 public:
   explicit CFavouriteContextMenuAction(uint32_t label) : CStaticContextMenuAction(label) {}
   bool IsVisible(const CFileItem& item) const override;
-  bool Execute(const CFileItemPtr& item) const override;
+  bool Execute(const std::shared_ptr<CFileItem>& item) const override;
```

Comment:
Please add `#include <memory>`to this header.

Reply:
done